ros_gz_sim_demos/config/air_pressure.yaml

```diff
@@ -0,0 +1,6 @@
+# Air pressure bridge configuration.
+- topic_name: "air_pressure"
+  ros_type_name: "sensor_msgs/msg/FluidPressure"
+  gz_type_name: "gz.msgs.FluidPressure"
+  lazy: true
+  direction: GZ_TO_ROS
```

Comment:
```suggestion
  direction: GZ_TO_ROS
- ros_topic_name: "clock"
  gz_topic_name: "/clock"
  ros_type_name: "rosgraph_msgs/msg/Clock"
  gz_type_name: "gz.msgs.Clock"
  direction: GZ_TO_ROS
```

Reply:
still pending

Reply:
Do we need to bridge `/clock` if we don't use `use_sim_time`?

Reply:
Yes, this is required by `rqt_topic`